test/e2e/network_segmentation.go

```diff
@@ -172,6 +176,8 @@ var _ = Describe("Network Segmentation", func() {
 						}, metav1.CreateOptions{})
 						Expect(err).NotTo(HaveOccurred())
 						defer func() {
+							err := cleanupPods(cs, defaultNetNamespace)
+							Expect(err).NotTo(HaveOccurred())
```

Comment:
nit: I would move it closer to where pod is create, line 220

Reply:
while that would work because defer is a stack I thought it would look cleaner if it was done as part of cleaning up the namespace in general. It does not matter if there are actually pods in the namespace or not since we create it it should try deleting all the pods.